cpp/tests/grouped_rolling/grouped_rolling_test.cpp

```diff
@@ -112,8 +112,8 @@ class GroupedRollingTest : public cudf::test::BaseFixture {
                                 size_type min_periods)
   {
     size_type num_rows = input.size();
-    std::vector<cudf::size_type> ref_data(num_rows);
-    std::vector<bool> ref_valid(num_rows);
+    thrust::host_vector<cudf::size_type> ref_data(num_rows);
```

Comment:
Why is it necessary to change to `host_vector` from `vector`?

Reply:
[vector of bool](https://howardhinnant.github.io/onvectorbool.html)

Reply:
Yes, this caused me a lot of headaches. Probably one of C++ biggest mistakes.

Reply:
Right, I remember the discussion now. We'll have to watch out for this in new tests that are added.